addons/hr/views/hr_employee_public_views.xml

```diff
@@ -134,46 +134,50 @@
                                         <strong class="o_kanban_record_title">
+                                            <div class="float-end">
+                                                <t t-if="record.show_hr_icon_display.raw_value">
+                                                    <div class="o_employee_availability d-flex flex-row align-items-start">
+                                                        <!-- Employee is present/connected and it is normal according to his work schedule  -->
+                                                        <span t-if="record.hr_icon_display.raw_value == 'presence_present'"
+                                                              class="fa fa-2x fa-circle text-success" name="presence_present" role="img"
+                                                              aria-label="Present" title="Present"
+                                                        />
+                                                        <!-- Employee is not present/connected and it is normal according to his work schedule -->
+                                                        <span t-if="record.hr_icon_display.raw_value == 'presence_absent'"
+                                                              class="fa fa-2x fa-circle-o text-warning" name="presence_absent" role="img"
+                                                              aria-label="Absent" title="Absent"
+                                                        />
+                                                        <!-- Employee is connected but according to his work schedule, he should not work for now  -->
+                                                        <span t-if="record.hr_icon_display.raw_value == 'presence_absent_active'"
+                                                              class="fa fa-2x fa-circle-o text-success" name="presence_absent_active" role="img"
+                                                              aria-label="Present but not active" title="Present but not active"
+                                                        />
+                                                        <!-- Employee is not here but according to his work schedule, he should be connected -->
+                                                        <span t-if="record.hr_icon_display.raw_value == 'presence_to_define'"
+                                                              class="fa fa-2x fa-circle text-muted" name="presence_to_define" role="img"
+                                                              aria-label="To define" title="To define"
+                                                        />
+                                                    </div>
+                                                </t>
+                                            </div>
                                             <field name="name" placeholder="Employee's Name"/>
```

Comment:
You can move this field before the float-end div above, makes more sense with read direction